parachains/runtimes/assets/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -524,3 +552,102 @@ fn foreign_pallet_has_correct_local_account() {
 	let address = Ss58Codec::to_ss58check_with_version(&account, polkadot);
 	assert_eq!(address, "13w7NdvSR1Af8xsQTArDtZmVvjE8XhWNdL4yed3iFHrUNCnS");
 }
+
+/// All configuration related to bridging
+pub mod bridging {
```

Comment:
Same comments/questions as with Kusama.

Reply:
docs applied and question answered here: https://github.com/paritytech/cumulus/pull/2762#discussion_r1276196879